test/unit/services/platform-test.js

```diff
@@ -74,4 +75,54 @@ describe('Platform', function() {
 
   });
 
+  describe('isVirtualizationEnabled', function(){
+
+    describe('on mac', function() {
+      it('should return true if virtualization check shell script returns true');
+    });
+
+    describe('on linux', function() {
+      it('should return true if virtualization check shell script returns true');
+    });
+
+    describe('on windows', function(){
+
+      it('should return promise resolved to true if powershell script returns `True` in stdout', function() {
+        let stub = sandbox.stub(Platform, 'getOS').returns('win32');
+        let stubs = sandbox.stub(child_process, 'exec').yields(undefined, "True", undefined);
+        return Platform.isVirtualizationEnabled()
+        .then(() => {
+          expect(stubs).to.be.calledOnce;
```

Comment:
it should test the statement in it() call first parameter. Here and later on it should look similar to:
```JavaScript
...
return Platform.isVirtualizationEnabled().then((result) => {
   expect(result).to.be.true;
});
```